code/style.css

```diff
@@ -1,4 +1,215 @@
-html {
-  background: #0025ff;
-  color: #fff;
+* {
+  box-sizing: border-box;
+  font-family: 'Courier New', Courier, monospace;
+  margin: 0;
+  line-height: 1.6;
 }
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  animation: header 2s ease-in-out ;
+  margin-bottom: 10px;  
+  border-bottom: 2px solid black;
+  animation: header 2s ease-in-out;
+}
+
+.logo {
+  height: 100px;
+  transform: rotateY(180deg);
+}
+
+h1 {
+  text-transform: uppercase;
+}
+
+.navigation a {
+  color: black;
+  margin: 0 10px;
+  font-size: 1.2em;
+  text-decoration: none;
+  background-color: lightgray;
+}
+
+
+a:visited {
+  color: black;
+}
+
+a:active {
+  color: red;
+}
+
+.navigation a:hover, .links a:hover {
+  text-decoration: underline;
+  color: red;
+  
+}
+/* big news */
+
+.big-news {
+  display: grid;
+  grid-template-columns: repeat(2, 50%);
+  margin: 30px 5px;
+  animation: start 2s ease-in-out;
+}
+
+.big-news-img {
+  max-width: 90%;
+  box-shadow: 4px 6px 6px rgba(0, 0, 0, 0.8);
+  margin: 5px 0 10px 5px;
+}
+
+.big-news-img:hover {
+  opacity: 0.7;
+}
+
+.intro {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  text-align: center;
+  padding: 10px;
+}
+
+/* small news cards */ 
+
+.container {
+  background-color: lightgray;
+  display: grid;
+  grid-template-columns: auto auto auto auto;
+  grid-template-rows: auto auto;
+  justify-items: center;
+  border: 2px solid black;
+  margin: 10px;
+}
+
+.small-news {
+  background-color: #fff;
+  max-width: 90%;
+  box-shadow: 2px 6px 4px rgba(0,0,0,0.8);
+  padding: 10px 5px 10px 0;
+  margin: 30px 0;
+  text-align: center;
+  line-height: 1.6;
+}
+
+.small-news > .small-image  {
+  width: 100%;
+  padding: 10px 20px 30px 20px;
+}
+
+.small-news p {
+  padding-top: 10px;
+}
+
+/* footer */
+
+footer {
+  text-align: center;
+  margin-top: 30px;
+  border-top: 3px solid black;
+  padding-top: 10px;
+}
+
+.links img {
+  padding-top: 5px;
+  filter: grayscale(30%);
+}
+
+.links a {
+  color: black;
+}
+
+
+/* animations */
+
+@keyframes header {
+  0% {
+      transform: translateY(-100%);
+      opacity: 0;
+  }
+ 
+  100% {
+      transform: translateY(0%);
+      opacity: 1;
+  }
+}
+
+@keyframes start {
```

Comment:
Same thing as before - add a comment on how it works would help (maybe also for yourself and remember?)

Reply:
Will add comment!